style.css

```diff
@@ -217,6 +243,18 @@ a:active {
   width: 100%;
 }
 
+.inner-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  min-height: calc(100vh - 44px);
+}
```

Comment:
- [ ] Let's make sure in desktop view we don't have these spaces between content to match the Figma design

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/155604286-5f738baf-bb8c-4687-a4e4-64a35a10410b.png) | ![image](https://user-images.githubusercontent.com/87689547/155604332-81fdbe5b-4284-4ef2-bf1e-6f731f0e486e.png)